company-packs/all.json

```diff
@@ -0,0 +1,58 @@
+[
+    {
+        "slug": "social-media",
+        "type": "choose",
+        "companies": [
+            "discord",
+            "facebook",
+            "google",
+            "instagram",
+            "linkedin",
+            "microsoft",
+            "pinterest",
+            "reddit",
+            "snapchat",
+            "telegram",
+            "tiktok",
+            "twitter",
+            "whatsapp",
+            "zoom"
+        ]
+    },
+    {
+        "slug": "entertainment",
+        "type": "choose",
+        "companies": [
+            "apple",
+            "deezer",
+            "disney",
+            "google",
+            "netflix",
+            "soundcloud",
+            "spotify",
+            "twitch"
+        ]
+    },
+    {
+        "slug": "finance",
+        "type": "choose",
+        "companies": [
+            "mastercard-eu",
+            "patreon",
+            "visa-eu"
+        ]
+    },
+    {
+        "slug": "telecommunication",
+        "type": "choose",
+        "companies": [
+            "google",
+            "heinlein-support-de",
+            "microsoft",
+            "proton-technologies",
+            "tutanota",
+            "verizon-media",
+            "yandex"
```

Comment:
See the commit message for the reasoning. The global list is mostly based on the Wikipedia list of webmailers.

What would you use instead?

Reply:
Heinlein Support is globally a big player in Webmailing? Not sure this is true. Maybe they are relevant in Germany. Globally, I have to say, Google, Microsoft and Yandex are they only really relevant ones. Why is Yahoo missing? Maybe we should leave out email providers in here.

On another note: I think it would be good not to select all runs entries of these companies, since not all are specific for the pack purpose. But that would require a more complicated schema.

Reply:
> On another note: I think it would be good not to select all runs entries of these companies, since not all are specific for the pack purpose. But that would require a more complicated schema.

That would also be confusing. If I add Google through the pack, it's not unreasonable to expect all of Google's activities to be covered. And I'm not sure whether this issue is really specific to the packs. Rather, we should make it clear that runs entries can be deselected in general. But I'm not sure if there's a better way for that than what we're currently doing already without overloading the user.

Reply:
And, at the end of the day, I think it's better if the user unexpectedly requests data for more services than not requesting it for all services they thought they were requesting for.

Reply:
Ah, I see. Some slugs are confusing :D

> And, at the end of the day, I think it's better if the user unexpectedly requests data for more services than not requesting it for all services they thought they were requesting for.

Yes, I agree, of course. And I guess you are right that users would probably assume we do this.